utils/codegen/codegen-rust/asn1ToRosMsg.py

```diff
@@ -127,7 +127,7 @@ def main():
                     msg = file.read()
 
                 type = os.path.basename(f).split('.')[0]
-                raw_def = asn1_raw[type]
+                raw_def = asn1_raw[type] if type in asn1_raw else ""
```

Comment:
Some raw message type definitions are apparently not extracted correctly, e.g., `ConnectedTrajectory`.

Reply:
There is a circular dependency in the headers - e.g.

```
--- stderr: etsi_its_mapem_ts_coding                                                                  
In file included from /home/james/zio/src/etsi_its_messages/etsi_its_coding/etsi_its_mapem_ts_coding/include/etsi_its_mapem_ts_coding/mapem_ts_SignalHeadLocation.h:15,
                 from /home/james/zio/src/etsi_its_messages/etsi_its_coding/etsi_its_mapem_ts_coding/include/etsi_its_mapem_ts_coding/mapem_ts_SignalHeadLocationList.h:44,
                 from /home/james/zio/src/etsi_its_messages/etsi_its_coding/etsi_its_mapem_ts_coding/include/etsi_its_mapem_ts_coding/mapem_ts_MapData-addGrpC.h:46,
                 from /home/james/zio/src/etsi_its_messages/etsi_its_coding/etsi_its_mapem_ts_coding/include/etsi_its_mapem_ts_coding/mapem_ts_RegionalExtension.h:18,
                 from /home/james/zio/src/etsi_its_messages/etsi_its_coding/etsi_its_mapem_ts_coding/include/etsi_its_mapem_ts_coding/mapem_ts_AdvisorySpeed.h:63,
                 from /home/james/zio/src/etsi_its_messages/etsi_its_coding/etsi_its_mapem_ts_coding/src/mapem_ts_AdvisorySpeed.c:8:
/home/james/zio/src/etsi_its_messages/etsi_its_coding/etsi_its_mapem_ts_coding/include/etsi_its_mapem_ts_coding/mapem_ts_NodeOffsetPointXY.h:53:17: error: unknown type name ‘mapem_ts_Reg_NodeOffsetPointXY_
t’                                                                                                    
   53 |                 mapem_ts_Reg_NodeOffsetPointXY_t         regional;                            
      |                 ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~                                              
gmake[2]: *** [CMakeFiles/etsi_its_mapem_ts_coding.dir/build.make:1378: CMakeFiles/etsi_its_mapem_ts_coding.dir/src/mapem_ts_AdvisorySpeed.c.o] Error 1
```

The file `mapem_ts_NodeOffsetPointXY.h` is included a second time by an import that occurs before the `struct` is defined, so it never gets to the definition and it fails.

I'm not sure how to break apart the circular import. Switching to the new `release2` branch of the `is_ts103301` repo didn't help - it just broke it elsewhere.